pkg/apis/serving/v1alpha1/revision_lifecycle.go

```diff
@@ -251,6 +257,12 @@ func RevisionContainerExitingMessage(message string) string {
 	return fmt.Sprintf("Container failed with: %s", message)
 }
 
+// RevisionSecretUnavailableMessage constructs the status message if a given
+// secret cannot be get by name
+func RevisionSecretUnavailableMessage(secretName string, message string) string {
```

Comment:
Does this really need to be exposed more than locally?

```suggestion
func revisionSecretUnavailableMessage(secretName string, message string) string {
```